.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -44,12 +46,9 @@ jobs:
     - name: Set up Cloud SDK
       uses: google-github-actions/setup-gcloud@v2
 
-    - name: Checkout code
-      uses: actions/checkout@v4
-
     - name: Generate offline vulnerability bundle
-      run: |
-        bash .github/workflows/scripts/scanner-update-offline-bundle.sh "$ROX_PRODUCT_VERSION"
+      run: .github/workflows/scripts/scanner-update-offline-bundle.sh "$ROX_PRODUCT_VERSION"
+      shell: bash
```

Comment:
Nitpick: Setting `shell` here does not make a difference because the script already shebangs to `/bin/bash`.